src/pages/patterns/global-header/index.mdx

```diff
@@ -0,0 +1,299 @@
+| ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| System  | System level processes include navigating to the main sections of a platform and accessing system utilities like notifications or settings. At this level a user can manage the attributes that apply to the whole system. |
+| Product | Product level processes include navigating within a product and accessing the product's core functions. At this level the user directly interacts with the product's main function.                                        |
+
+### Example matrix
+
+This matrix shows common system and product level processes that appear globally and locally.
+
+<Row>
+<Column colLg={12}>
+
+|                        | System processes                                                                                                                                   | Product processes                                                                                                                                                                                |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Global persistence** | Log in, notifications, profile access, dashboard link, setting time, billing information, user permissions, switching from one product to another. | Hamburger menu to toggle local nav. Show/hide application navigation, document thumbnails. Consistent placement of local actions in common between products for example: new file, save, cancel. |
+| **Local persistence**  | Detailed system-wide settings, such as user rights management, notification preferences.                                                           | Commenting in a text doc, navigating from one peer resource to another (for example from one container, data base, or document to another), selecting modalities within your product.            |
+
+</Column>
+</Row>
+
+## Configurations
+
+The UI shell is configurable to let a product or platform choose which shell components and configurations match their user and information requirements. Simple products have the flexibility of using a header or side panel and more complex products require a combination of UI shell components to accommodate the depth of their navigation.
+
+![Example the header and left nav being used for the same IA.](images/configurations.png)
+
+<Caption>
+  A simple product navigation can be achieved with either a header or side panel
```

Comment:
Is it possible to include real IBM product examples here?

Reply:
The only real examples that exist today are the IBM design and the Carbon sites. I had those in there earlier but was told to take them out. I could mock up some fake ones for now if that's what we want.

Reply:
Possible tweak to this text;

"Simpler navigation patterns have the flexibility of using either a header or side panel. More complex navigation needs can be accommodated with a combination of UI shell components."

Main intent is to talk about complexity of navigation versus complexity of product.